script/cross-validation

```diff
@@ -6,7 +6,7 @@ ACCEPTABLE_ERRORS = 48
 
 # Number of acceptable classification errors when using --all.
 # It should only be decreased.
-ACCEPTABLE_ERRORS_ALL = 669
+ACCEPTABLE_ERRORS_ALL = 671
```

Comment:
Note that this was 672 in master, it seems to have drifted because of new samples.